BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+ExchangeRateService service = new ExchangeRateService();
+
+// REST API to get exchange rates based on input currencies
+app.MapPost("/", service.GetExchangeRates);
+
+app.Run();
+
+public struct CnbRatesResponse
+{
+    [JsonPropertyName("rates")] public List<Rate> Rates { get; set; }
+}
+
+public struct Rate
+{
+    [JsonPropertyName("currencyCode")] public string CurrencyCode { get; set; }
+    [JsonPropertyName("rate")] public float RateValue { get; set; }
+}
+
+internal class ExchangeRateService
```

Comment:
For the class “ExchangeRateService”, it doesn’t seem in the correct place right? Let’s create folders which names “Services” and “IServices”. Why 2 folders??  One of this is for the service classes that we wrote and other one is for their interface. I can hear taht you’re saying “Why we need interface?? Just using a service also works.” Yes, you’re correct. But answer is “Loosely Coupled”.